nupic/regions/CLAClassifierRegion.py

```diff
@@ -263,6 +355,30 @@ def customCompute(self, recordNum, patternNZ, classification):
                                         infer = self.inferenceMode)
 
 
+  def getOutputValues(self, outputName):
+    """Return the dictionary of output values. Note that these are normal Python
+    lists, rather than numpy arrays. This is to support lists with mixed scalars
+    and strings, as in the case of records with categorical variables
+    """
+    return self._outputValues[outputName]
+
+
+  def getOutputElementCount(self, name):
+    """Returns the width of dataOut."""
+   
+    if name == "classificationResults":
+      return len(self.stepsList)
+    elif name == "probabilities":
+      if not self.numCategories:
```

Comment:
when would this be true? Is this for backwards compatibility? In that case, it should be updated in the deserialization to upgrade old version. Same for "actualValues" as well